.github/ISSUE_TEMPLATE/feature.md

```diff
@@ -0,0 +1,22 @@
+---
+name: Feature request
+about: Suggest a new feture or an improvement to existing functionality
+title: ''
+labels: type/enhancement, status/triage
+assignees: ''
+
+---
+
+**Description**
```

Comment:
Nitpick: Might want to use headers `####` to be more visually separating given that might use bold text in their responses.

Reply:
Good call to make them consistent.